### PR TITLE
Add jeremy.txt

### PR DESCRIPTION
**Pull request by OatmealSoldier**
I added a file called jeremy.txt

### CODE REVIEW

File: jeremy.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+I am Jeremy.
```
Comment (OatmealSoldier) [inline jeremy.txt]: That's me!

**Comment by SEStarkman** [comment]
looks great

File: jeremy.txt (hunk 1 above)
Comment (yiskylee) [inline jeremy.txt]: Could you also state your last name please? Otherwise I wouldn't accept the request.

**Comment by SEStarkman** [comment]
i wont accept until you provide your credit card info

File: jeremy.txt (hunk 1 above)
Comment (kkimelman) [inline jeremy.txt]: Can you list what grade you are in college? ex. freshman sophomore ... etc

Comment (emersonboyd) [inline jeremy.txt]: Can you add a new line with the date please?

**Pull request merged**